Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentController.cpp

```diff
@@ -148,6 +152,100 @@ namespace MiniAudio
+        OnConfigurationUpdated();
+    }
+
+    float MiniAudioPlaybackComponentController::GetInnerAngleInDegrees() const
+    {
+        return m_config.m_innerAngleInDegrees;
+    }
+
+    void MiniAudioPlaybackComponentController::SetInnerAngleInDegrees(float innerAngleInDegrees)
+    {
+        m_config.m_innerAngleInDegrees = innerAngleInDegrees;
+        m_config.m_innerAngleInRadians = AZ::DegToRad(m_config.m_innerAngleInDegrees);
+        OnConfigurationUpdated();
+    }
+
+    float MiniAudioPlaybackComponentController::GetOuterAngleInRadians() const
+    {
+        return m_config.m_outerAngleInRadians;
+    }
+
+    void MiniAudioPlaybackComponentController::SetOuterAngleInRadians(float outerAngleInRadians)
+    {
+        m_config.m_outerAngleInRadians = outerAngleInRadians;
+        m_config.m_outerAngleInDegrees = AZ::RadToDeg(m_config.m_outerAngleInRadians);
+        OnConfigurationUpdated();
+    }
+
+    float MiniAudioPlaybackComponentController::GetOuterAngleInDegrees() const
+    {
+        return m_config.m_outerAngleInDegrees;
+    }
+
+    void MiniAudioPlaybackComponentController::SetOuterAngleInDegrees(float outerAngleInDegrees)
+    {
+        m_config.m_outerAngleInDegrees = outerAngleInDegrees;
+        m_config.m_outerAngleInRadians = AZ::DegToRad(m_config.m_outerAngleInDegrees);
+        OnConfigurationUpdated();
+    }
+
+    float MiniAudioPlaybackComponentController::GetOuterVolume() const
+    {
+        return m_config.m_outerVolume;
+    }
+
+    void MiniAudioPlaybackComponentController::SetOuterVolume(float outerVolume)
+    {
+        m_config.m_outerVolume = AZ::GetClamp(outerVolume, 0.f, 100.f);
+        OnConfigurationUpdated();
+    }
+
+    bool MiniAudioPlaybackComponentController::GetFixedDirecion() const
+    {
+        return m_config.m_fixedDirection;
+    }
+
+    void MiniAudioPlaybackComponentController::SetFixedDirecion(const bool& fixedDirection)
+    {
+        m_config.m_fixedDirection = fixedDirection;
+    }
+
+    float MiniAudioPlaybackComponentController::GetDirectionalAttenuationFactor() const
+    {
+        return ma_sound_get_directional_attenuation_factor(m_sound.get());
+    }
+
+    void MiniAudioPlaybackComponentController::SetDirectionalAttenuationFactor(float directionalAttenuationFactor)
+    {
+        m_config.m_directionalAttenuationFactor = directionalAttenuationFactor;
+        OnConfigurationUpdated();
+    }
+
+    AZ::Vector3 MiniAudioPlaybackComponentController::GetDirection() const
+    {
+        ma_vec3f direction = ma_sound_get_direction(m_sound.get());
+        return AZ::Vector3(direction.x, direction.y, direction.z);
+    }
+
+    void MiniAudioPlaybackComponentController::SetDirection(const AZ::Vector3& direction)
+    {
+        m_config.m_direction = direction;
+        AZ::Transform worldTm = AZ::Transform::CreateIdentity();
+        AZ::TransformBus::EventResult(worldTm, m_entityComponentIdPair.GetEntityId(), &AZ::TransformBus::Events::GetWorldTM);
```

Comment:
worldTM doesn't appear to be used here

Reply:
Removed it in 87d1400dfa20a3850ea7f1a1a4712de7730b4318